group/iab/IAB_Decisions.md

```diff
@@ -54,66 +58,104 @@ for the IAB chair to determine whether it will be held open or secret.
+occur at different times to accomodate IAB members in different timezones.
+Note that lack of attendence by IAB members due to vacations or choosing not to
+attend a meeting will not delay being able to hold the vote, as long as meeting
+opportunities were provided.
+
+Any meeting on which a vote is discussed needs to either be a regularly scheduled
+meeting, or be an extraordinary meeting scheduled at least a week in advance.
+
+Discussions in meetings need to follow a regular procedure for adding agenda items
+to the meeting. Voting discussions should be added to a meeting agenda at least
+48 hours prior to the meeting, or earlier depending on the current meeting practice
+of the IAB.
+
+### Conducting votes
+
+Once the discussion opportunities (described above) have been provided, a vote
+can formally start. This can occur during a teleconference or in-person meeting
+directly after discussion, but unless all IAB members are present, will always
+also be started virtually.
+
+Voting typically occurs on a private email list that includes all voting members,
+but may be conducted using other virtual tools.
+
+### Concluding votes
+
+A vote will typically stay open until a position has been recorded for every
+single voting member, even after a majority has been reached. This ensures
+every voice can be heard. Note that it is reasonable for any voting member to
+request a few days to further review available information before voting.
+
+A vote can conclude in either of the following conditions:
+
+- All voting members have submitted their votes
+- Seven days have elapsed since the vote was started, and the requirements
+  from RFC 2850 have been met (specifically, that seven full members concur
+  and there are no more than two dissents)
+
+If these conditions cannot be met within seven days, the IAB chair can
+choose to either leave the vote open, or cancel the vote.
 
 ## Alternative Process
 
 The IAB can use alternative voting processes. In order to do that, the IAB
-chair must announce the alternative voting rules to the IAB at least one week
-before the voting takes place. If any voting member of the IAB objects to the
-alternative voting rules, the resolution of the conflict will be addressed
-following usual RFC 2850 guidelines; i.e., if more than two voting members
-object to the alternative voting process, the vote reverts to the Normal
-Process.
+chair must announce the alternative voting rules to the IAB. This
+announcement should usually occur at least one week before the voting takes
+place. Per RFC 2950, if more than two voting member of the IAB objects to the alternative voting rules,
+the vote reverts to the Normal Process.
+
+Unanimous results should be strongly preferred in votes using an alternative
+process. Particularly, if the vote is run on a shortened timeline (such as
+having less than a week of advanced notice), unanimity should be required.
```

Comment:
In the first sentence, it says preferred, but in the 2nd, it says required. Can this be tightened up?

Reply:
The distinction for me was that if the alternative process is done to be faster, it should require unanimity. However, some hypothetical alternative process could also be a longer process, and may be OK to still have non-unanimity.

Reply:
```suggestion
having less than a week of advanced notice), unanimity from those who respond
should be required.
```